plugins/sap_hana_logs.yaml

```diff
@@ -0,0 +1,62 @@
+version: 0.0.1
+title: SAP HANA
+description: Log parser for SAP HANA
+parameters:
+  - name: file_path
+    description: The directory to the SAP HANA trace logs. No trailing slash should be included.
+    type: "[]string"
+    default:
+      - "/usr/sap/*/HDB*/*/trace/*.trc"
+  - name: exclude
+    description: The directories to exclude for the SAP HANA trace logs.
+    type: "[]string"
+    default:
+      - "/usr/sap/*/HDB*/*/trace/nameserver_history*.trc"
+      - "/usr/sap/*/HDB*/*/trace/nameserver*loads*.trc"
+      - "/usr/sap/*/HDB*/*/trace/nameserver*unlaods*.trc"
+      - "/usr/sap/*/HDB*/*/trace/nameserver*executed_statements*.trc"
+  - name: start_at
+    description: Start reading file from 'beginning' or 'end'
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end
+
+template: |
+  receivers:
+    filelog:
+      include:
+          # {{ range $fp := .file_path }}
+             - '{{ $fp }}'
+          # {{end}}
+      exclude:
+          #  {{ range $fp := .exclude }}
+             - '{{ $fp }}'
+          # {{end}}
+      multiline:
+         line_start_pattern: '\[\d+\]{[^}]+}\[[^\/]+\/[^\]]+\] \d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}\.\d+ [^ ]+ [^ ]+\s+[^ ]+ :'
+      start_at: {{ .start_at }}
+      attributes:
+         log_type: sap_hana
+      operators:
+          - id: sap_hana_parser
+            type: regex_parser
+            regex: '^\[(?P<thread_id>\d+)\]{(?P<connection_id>[^}]+)}\[(?P<transaction_id>[^\/]+)\/(?P<update_transaction_id>[^\]]+)\] (?P<timestamp>\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}\.\d+) (?P<sap_severity>[^ ]+) (?P<component>[^ ]+)\s+(?P<source_file>[^ ]+) : (?P<message>.*)'
+            timestamp:
```

Comment:
do we need to do something for timezone for this one?

Reply:
I am not entirely certain.

Reply:
Heard and added